### DISCUSSION TITLE
How to add another php version in Laragon on Windows PC

### DISCUSSION THREAD

**Discussion by rafatriolo** [leokhoa/laragon#479] [General]
Hi, since @leokhoa has moved the discussion to here, I want to contribute by sharing how to add another php version in laragon that is running on windows pc. Just go to this url: "https://windows.php.net/downloads/releases/" and then dowload the version that you want, unzip and put the folder on laragon "bin/php" folder. Now this new version will appear in laragon´s menu for you to choose. 
I hope it helps someone. 

Tks

**Comment by Pen-y-Fan** [comment]
I've written a blog [How to update the PHP version in Laragon](https://pen-y-fan.github.io/2023/01/15/how-to-update-the-php-version-in-laragon/) with step by step guide. I hope this will help.

**Comment by megaaleksandar** [comment]
Hi everyone, I just want to update the link with the all "old" releases here [PHP Archives](https://windows.php.net/downloads/releases/archives/) and you should download the {php-version}-WIN32-VC{version}-x{architecture}.zip

**Comment by boranbar** [comment]
Don't forget to **Remove Laragon From PATH** and **Add Laragon to PATH** again when switching between PHP versions.

**Reply by Nipodemos** [reply]
thanks for the reminder

**Reply by aaronedev** [reply]
which path do you mean?

**Reply by Pen-y-Fan** [reply]
See my blog [How to set up Laragon on a new Windows computer (part 6) - Add to path](https://pen-y-fan.github.io/2023/01/05/how-to-set-up-laragon-on-a-new-windows-computer-part-6/).

If you use another terminal, than the commander terminal built into Laragon, you need to switch PATH to keep them in sync.

**Reply by aaronedev** [reply]
ah ye i see thanks

**Comment by dawoodPK** [comment]
I want to add php 8.3.22 so it worked for following steps:

my PC requirements:
- x64
- laragon 6.0

**step # 1**
downloaded "php-8.3.22-nts-Win32-vs16-x64.zip" from [PHP offical releases](https://windows.php.net/downloads/releases/)
NOTE: it must be **NTS** version
[php-8.3.22-nts-Win32-vs16-x64.zip](https://windows.php.net/downloads/releases/php-8.3.22-nts-Win32-vs16-x64.zip)

**step # 2** 
copy the downloaded zip file to the "laragon\bin\php\". (see screenshot to quickly go to folder laragon installation folder)
![image](https://github.com/user-attachments/assets/0367cc9f-eb3e-49da-8a7e-1068d34ffbad)
then go to "bin" and "php" folders . then paste file

**step # 3** 
extract zip to folder "php-8.3.22-nts-Win32-vs16-x64" folder in "php" folder.
now confirm by opening "php-8.3.22-nts-Win32-vs16-x64" folder you see file like following img.

![image](https://github.com/user-attachments/assets/76bbaa7e-c50e-48c3-8b76-cf5d9afc6a3d)

 if not then fix it first. 

**step # 4** 
on windows search "Environment variable" and follow screenshot

![image](https://github.com/user-attachments/assets/43abfa4a-138a-4552-8171-431c0af031e5)

then update/add this new php verison path to "path" configuration and save.
open command prompt and run  ```php -v```
you will see "PHP 8.3.22 ....... ...... " it means php is working. 

**step # 5** 
stop all Laragon services (stop button). and wait until all stoped. 
now change PHP version from laragan menu as show in image. 

![image](https://github.com/user-attachments/assets/f7a818a2-d544-43ac-be6d-148a6dfcb426)


Start all laragon services again. 
visit any project page. 

You are done!

If you face any issue, tell me in comment please.

**Comment by Jonwork09** [comment]
thanks'
